packages/peregrine/lib/Router/resolveUnknownRoute.js

```diff
@@ -89,22 +89,23 @@ function remotelyResolveRoute(opts) {
  * @returns {Promise<{type: "PRODUCT" | "CATEGORY" | "CMS_PAGE"}>}
  */
 function fetchRoute(opts) {
+    const query = `query ResolveURL($url: String!) {
+        urlResolver(url: $url) {
+            type
+            id
+        }
+    }`;
+
     const url = new URL('/graphql', opts.apiBase);
+    url.searchParams.set('query', query);
```

Comment:
Since this query is made outside the apollo client it does not get stripped of ignored characters.

Reply:
Yeah, I kind of figured it didn't matter here, since the total length of this query is based on the URL of the page.

Want me to change it up?

Reply:
Not really, just pointing it out :) If/When we move url resolver to go through ApolloClient it won't matter.